### PR TITLE
HW27-1

### PR DESCRIPTION
**Pull request by skayKatya**

### CODE REVIEW

File: state/user1.json
Comment (sttaran) [inline state/user1.json:1]: add it to gitignore

Comment (sttaran) [reply state/user1.json:1]: and remove it

File: tests/HW27-1/HW27-1-profileInfo.spec.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+        await page.route("/api/users/profile", async route =>{
+        await route.fulfill({
+                  status: 200,
+                  json: mockedProfileinfo
+                })
+            })
+
+            await page.reload()
+            await page.pause()
```
Comment (sttaran) [inline tests/HW27-1/HW27-1-profileInfo.spec.js:30]: check if value was mocked on UI

**Review by sttaran — CHANGES_REQUESTED** [review]

File: tests/HW27-1/HW27-1-profileInfo.spec.js (hunk 1 above)
Comment (skayKatya) [reply tests/HW27-1/HW27-1-profileInfo.spec.js:30]: after adding
 await page.reload()
 await page.pause() -> I saw new values 
 "name": "test",
  "lastName": "Nework"

Comment (skayKatya) [reply tests/HW27-1/HW27-1-profileInfo.spec.js:30]: 2024-08-16_12h29_14

File: state/user1.json
Comment (skayKatya) [reply state/user1.json:1]: done

**Pull request merged**